p4src/tna/fabric_tna.p4

```diff
@@ -76,9 +75,9 @@ control FabricIngress (
         if (!fabric_md.skip_next) {
             pre_next.apply(hdr, fabric_md);
         }
-        acl.apply(hdr, fabric_md, ig_intr_md, ig_dprsr_md, ig_tm_md);
+        acl.apply(hdr, fabric_md, ig_intr_md, ig_dprsr_md, ig_tm_md, (FabricPortId_t)ig_intr_md.ingress_port);
```

Comment:
Yes, this worked. I removed this for most of the control blocks. But the StatsIngress and StatsEgress are still using it since they already use this way.

Reply:
Looks good. Thank you.